README.md

```diff
@@ -267,21 +240,21 @@ I would like to thank...
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
+- **[How_to_contribute_thhis_proyect?]**
 
-  - [Answer_1]
+  - [You_can_do_it_in_https://github.com/AztkMarty]
 
-- **[Question_2]**
+- **[How_to_run_the_page?]**
 
-  - [Answer_2]
+  - [You_can_run_here_https://github.com/AztkMarty]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is licensed under the Apache License 2.0. and any other open-source.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
```

Comment:
- [ ] kindly consider customizing this section by removing those comments from the template given by Microverse to make your readme more professional, there  were given to guide you, and also consider adding a link to your MIT license you may refer on this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to do so